packages/eds-core-react/src/components/Banner/Banner.tsx

```diff
@@ -7,12 +7,14 @@ import {
 } from 'react'
 import styled, { css, ThemeProvider } from 'styled-components'
 import { spacingsTemplate, useToken } from '@equinor/eds-utils'
+import { Paper } from '../Paper'
+import type { Elevations } from '@equinor/eds-tokens'
 import { enabled as bannerToken } from './Banner.tokens'
 import { Divider } from '../Divider'
 import { BannerIcon } from './BannerIcon'
 import { useEds } from '../EdsProvider'
 
-const StyledBanner = styled.div``
+type allowedElevations = keyof Pick<Elevations, 'none' | 'raised' | 'overlay'>
```

Comment:
This is the type that users will see in their IDE, so maybe a nicer word, `availableElevations` ? Its also a type which means it should start with an Uppercase.

Reply:
Changing it to `AvailableElevations`